tests/filecheck/runner/riscv.mlir

```diff
@@ -0,0 +1,24 @@
+// RUN: xdsl-run %s
```

Comment:
can you add the filecheck bit, please?

Reply:
That would indeed be quite useful to have

Reply:
Fixed!